### ISSUE TITLE
[Archive's Request] warosu.org & archived.moe

### ISSUE THREAD

**Issue by darlopvil** [TrixiEther/Dashchan-Extensions#31]
These two archive sites are used a lot on 4chan, but there's no extension for them.

**Comment by TrixiEther** [comment]
@darlopvil archived.moe is done:
https://github.com/TrixiEther/Dashchan-Meta/raw/master/update/package/DashchanArchivedMoe.apk

You can download the module apk file. If you use DashchanFork, then the module is in the list for installation, plus, updates of new modules are supported in case of fixes

**Comment by darlopvil** [comment]
Thanks for your big effort. Didn't know of your Dashchan-Meta repository until now. I like how you've listed every apk in a clean and simple way. 🤝

**Comment by TrixiEther** [comment]
@darlopvil the main merit of the original author, I just made a fork and try to develop it further :)

**Comment by darlopvil** [comment]
Hello what's up,

I've seen you have included this issue in the "Todo" list (project view), I was shocked by all the issues still placed in there. Since I think I'm the only one asking for this archive thing and also, it isn't a life-threatening thing; I just wanted to tell you that it isn't more important than any other issue. Just do it at your one peace, no need to rash.

Thanks for your hard work, greetings 🤝

**Comment by TrixiEther** [comment]
@darlopvil "In Progress" - because one of the modules has already been made. I don’t think that I have such a serious development here, rather I just test the features of the github along the way and just use Issues as reminders so as not to forget what else I can play with :)

**Comment by TrixiEther** [comment]
@darlopvil hi!

Warosu module: https://github.com/TrixiEther/Dashchan-Meta/raw/master/update/package/DashchanWarosu.apk

At the moment, this is the first release, unlike the first module, I had to completely rewrite the parsing logic, because I found that the site is arranged quite differently. For this reason, it is possible that there will be some incorrect display. If you find something similar, then just create a new Issue, it's better if there is a screenshot of the problem and a link to a non-working thread...

What is known so far:

1. Most likely, the display of tripcodes does not work
2. I'm not sure that spoilers are displayed correctly

It’s problematic to do this right away, because I need a copy of such a thread ... but I searched all day and never found it :)

**Comment by darlopvil** [comment]
Came here as fast as I got the notification, i couldn't believe it, it was finally released!

 I've just tested it with a few threads, and it works pretty well. I've also checked that Japanese characters are shown and also erased posts, that'll make it for me.

Thanks for your hard work, I really appreciate it a lot. Well, me and anyone who uses this in the future. Blessings! 🤝🙏

> If you find something similar, then just create a new Issue

🫡
